kernel/tests/dv.rs

```diff
@@ -17,7 +17,7 @@ fn dv_table() -> Result<(), Box<dyn std::error::Error>> {
     let snapshot = table.snapshot(&engine, None)?;
     let scan = snapshot.into_scan_builder().build()?;
 
-    let stream = scan.execute(&engine)?;
+    let stream = scan.execute(&engine)?.map(Result::unwrap);
```

Comment:
Personally this looks a bit odd to me and I would prefer below to do
```rust
  let res = res?;
```
But I think it's more of an aesthetic choice so if you prefer this that's fine :)